packages/speech/src/index.js

```diff
@@ -364,13 +364,21 @@ Speech.formatResults_ = function(resultSets, verboseMode) {
  *     property, a `confidence` score from `0` - `100`, and an `alternatives`
  *     array consisting of other transcription possibilities.
  *
+ * Note that speech API sets the limits for the audio duration.
+ * See [Content Limits]{@link https://cloud.google.com/speech/limits#content} for the details.
+ *
  * @resource [StreamingRecognize API Reference]{@link https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.Speech.StreamingRecognize}
  * @resource [StreamingRecognizeRequest API Reference]{@link https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.StreamingRecognizeRequest}
+ * @resource [Content Limits]{@link https://cloud.google.com/speech/limits#content}
  *
  * @param {object} config - A `StreamingRecognitionConfig` object. See
  *     [`StreamingRecognitionConfig`](https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.StreamingRecognitionConfig).
  * @param {boolean=} config.verbose - Enable verbose mode for a more detailed
  *     response. See the examples below. Default: `false`.
+ * @param {number=} config.timeout - Customize the timeout number in
+ *     milliseconds for underlying gRPC call. This will affect the input audio
+ *     duration; longer timeout needs to be specified when the audio input is
+ *     longer. By default, it will use the timeout enough for 60 seconds audio.
```

Comment:
Sorry for missing the connection. Yes, my thought is that we should state here so the user has a point of reference for when they have input longer than 60 seconds. I will add this in for us real quick though, thanks!